index.html

```diff
@@ -27,72 +29,67 @@
-
-      ga('create', 'UA-56586105-1', 'auto');
-      ga('require', 'displayfeatures');
-      ga('send', 'pageview');
-    </script>
-
   </head>
   <body>
-          
-  <div class="container">
-          
-         <div class="col-xs-12 col-sm-12"> 
-                 <h1>Notation Capital</h1>
-    </div>
-
-    <h2 class="col-xs-12 col-sm-12">Purpose</h2>
-    
-        <p class="col-xs-12 col-sm-12">To help technical founders turn their obsessions into valuable products and companies.</p>
 
-    <h2 class="col-xs-12 col-sm-12">
-        <a href="https://medium.com/@NotationCapital/introducing-notation-capital-780a94b738d">Writing</a>
-    </h2>
-
-    <div class="imglinks">
-    <h2 class="col-xs-12 col-sm-12">Connect</h2>
+  <div class="container">
+      <div class="col-xs-12 col-sm-12">
+        <h1>Notation Capital</h1>
+      </div>
+      <h2 class="col-xs-12 col-sm-12">Purpose</h2>
+      <p class="col-xs-12 col-sm-12">To help technical founders turn their obsessions into valuable products and companies.</p>
+      <h2 class="col-xs-12 col-sm-12">
+          <a href="https://medium.com/@NotationCapital/introducing-notation-capital-780a94b738d">Writing</a>
+      </h2>
+      <div class="imglinks">
+        <h2 class="col-xs-12 col-sm-12">Connect</h2>
         <p class="col-xs-12 col-sm-12">
-          <a href="https://github.com/notationcapital"><img src="assets/images/github-48.png"></a>
-          <a href="https://angel.co/notation-capital"><img src="assets/images/angellist-48.png"></a>
-          <a href="https://instagram.com/NotationCapital/"><img src="assets/images/instagram-48.png"></a>
-          <a href="https://twitter.com/NotationCapital/"><img src="assets/images/twitter-48.png"></a>
-          <a href="mailto:connect@notationcapital.com"><img src="assets/images/email-48.png"></a>
-          <a href="https://goo.gl/maps/KXrTw"><img src="assets/images/ic_place_white_18dp.png"></a>
+          <a href="https://github.com/notationcapital">
+            <img class="social-img" src="assets/images/github-48.png" alt="Notation Capital - Github">
+          </a>
+          <a href="https://angel.co/notation-capital">
+            <img class="social-img" src="assets/images/angellist-48.png" alt="Notation Capital - AngelList">
+          </a>
+          <a href="https://instagram.com/NotationCapital/">
+            <img class="social-img" src="assets/images/instagram-48.png" alt="Notation Capital - Instagram">
+          </a>
+          <a href="https://twitter.com/NotationCapital/">
+            <img class="social-img" src="assets/images/twitter-48.png" alt="Notation Capital - Twitter">
+          </a>
+          <a href="mailto:connect@notationcapital.com">
+            <img class="social-img" src="assets/images/email-48.png" alt="Notation Capital - Email">
+          </a>
+          <a href="https://goo.gl/maps/KXrTw">
+            <img class="social-img" src="assets/images/ic_place_white_18dp.png" alt="Notation Capital - Map">
```

Comment:
map pin is actually 36x36 so this blows it up a little too large 
![icon_comp](https://cloud.githubusercontent.com/assets/41521/6427678/d84b5a9a-bf55-11e4-8a04-fe69922ef3ce.jpg)